amethyst_ui/src/cursor.rs

```diff
@@ -0,0 +1,59 @@
+//! Module containing the system managing the text editing cursor create, deletion and position.
+
+// TODO: Complete this and remove the logic from the UI Pass. Scheduled for completion after Transform2D.
+// File currently ignored because it is not added to lib.rs
+
+/// Tag component placed on the cursor of a text field being edited.
+pub struct TextEditingCursor;
```

Comment:
I feel like the naming for this and as well for the file could be better? `cursor` in `ui` sounds like the mouse cursor to me. Maybe just `TextCursor` is enough? Then the file could be `text_cursor.rs` as well. I think it make sense since you'd only ever have cursors in editable text anyways.